lib/features/modeling/Modeling.js

```diff
@@ -44,15 +45,16 @@ Modeling.$inject = [
 Modeling.prototype.getHandlers = function() {
   var handlers = BaseModeling.prototype.getHandlers.call(this);
 
-  handlers['element.updateProperties'] = UpdatePropertiesHandler;
```

Comment:
What happened here?

Reply:
Alphabetical sorting, it's a habit of mine but I didn't intend to commit it in the end, my bad.

Reply:
Fine to do these kind of things, just put them in an additional "chore" or "style" commit.